pipeline/lint.bash

```diff
@@ -15,7 +14,6 @@ charts_ignore_list=(
   "app!=kube-prometheus-stack"
   "app!=velero"
   "app!=metrics-server"
-  "app!=dex"
```

Comment:
Shouldn't we still exclude upstream charts?

Reply:
I tried, but the lint keep failing because some opensearch charts are depended on dex
t.ex https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/50-applications.yaml#L263

Reply:
I see, yeah the way newer versions of helmfile works with needs is not great. :sweat: